lib/paper_trail/record_trail.rb

```diff
@@ -148,11 +148,14 @@ def data_for_update
     # paper_trail-association_tracking
     def record_update_columns(changes)
       return unless enabled?
-      event = Events::Update.new(@record, false, false, changes)
+      data = Events::Update.new(@record, false, false, changes).data
+
+      # use AR default timestamp value instead of record.updated_at value
+      data.delete(:created_at)
```

Comment:
Why remove it here instead of removing the original assignment (`data[:created_at] =`) in `events/update.rb`?

Reply:
I want to specifically address `update_columns` method do not want to affect `build_version_on_update` (so the "regular" update event flow / core functionality).

Doing this in `Events::Update` would require me to add an additional parameter to the constructor. 

This felt like a cleaner and simpler solution that is just an adjustment for this specific method/scenario which I'm addressing, but I can change it if you think that would be better.

Reply:
Makes sense, thanks for the helpful explanation :)